akka-testkit-typed/src/main/scala/akka/testkit/typed/ExplicitlyTriggeredScheduler.scala

```diff
@@ -11,7 +11,7 @@ import scala.concurrent.duration.{ Duration, FiniteDuration }
 class ExplicitlyTriggeredScheduler(config: Config, log: LoggingAdapter, tf: ThreadFactory) extends akka.testkit.ExplicitlyTriggeredScheduler(config, log, tf) {
 
   @varargs
-  def expectNoMessageFor[T](duration: FiniteDuration, on: scaladsl.TestProbe[T]*): Unit = {
+  def expectNoMessageFor(duration: FiniteDuration, on: scaladsl.TestProbe[_]*): Unit = {
```

Comment:
Aah, I misread this to be on the probe, which seemed to magic, this is explicit enough for me! :) 👍